runtime/autoload/provider/node.vim

```diff
@@ -24,7 +44,7 @@ function! provider#node#Require(host) abort
   else
     let args = ['node']
 
-    if !empty($NVIM_NODE_HOST_DEBUG)
+    if provider#node#can_inspect() && !empty($NVIM_NODE_HOST_DEBUG)
```

Comment:
`!empty($NVIM_NODE_HOST_DEBUG)` should be tested first, that avoids a process invocation.